packages/ripple-nuxt-tide/lib/module.js

```diff
@@ -33,6 +33,33 @@ const nuxtTide = function (moduleOptions) {
     fileName: './request-id.js'
   })
   this.options.router.middleware.push('request-id')
+
+  // https://toor.co/blog/nuxtjs-smooth-scrolling-with-hash-links/
+  this.options.router.scrollBehavior = async (to, from, savedPosition) => {
```

Comment:
Would like to know what problem this fixed, I have a fix for the scroll already, is that issue still there?